examples/src/main/scala/zio/config/examples/ProgramExample.scala

```diff
@@ -19,77 +20,108 @@ object ProgramExample extends App {
   case class Live(config: Config.Service[ProgramConfig], spark: SparkEnv.Service)
       extends SparkEnv
       with Config[ProgramConfig]
+      with Blocking.Live
 
   override def run(args: List[String]): ZIO[Environment, Nothing, Int] = {
-    val pgm = for {
-      confService <- Config.fromEnv(programConfig)
-      session     <- ZIO.effect("sparkSession")
-      _           <- Application.execute.provide(Live(confService.config, new SparkEnv.Live(session)))
-    } yield ()
+    val pgm =
+      for {
+        configEnv <- Config.fromEnv(programConfig)
+        sparkEnv  <- SparkEnv.local("some-app")
+        _         <- Application.execute.provide(Live(configEnv.config, sparkEnv.spark))
+      } yield ()
 
-    ZIO.accessM[Environment](env => {
+    ZIO.accessM[Environment] { env =>
       pgm.foldM(
         fail => env.console.putStrLn(s"failed $fail") *> ZIO.succeed(1),
         _ => env.console.putStrLn(s"succeeded") *> ZIO.succeed(0)
       )
-    })
+    }
   }
 }
 
-// spark service
+//// stubs for the real Spark
+
+trait SparkSession {
+  def sql(value: String): DataFrame =
+    new DataFrame {}
+
+  def version: String =
+    "someVersion"
+}
+
+trait DataFrame
+
+////
```

Comment:
Let's remove `///` as we are not using anywhere else in the code.

Reply:
Let's also remove `DataFrame`. It's a bit too much detail of spark in this example. Those who are unfamiliar with spark may find it a bit confusing.,